search-books/src/main/java/nix/education/service/suggestions/solr/SolrBookAuthorSuggestionService.java

```diff
@@ -0,0 +1,59 @@
+package nix.education.service.suggestions.solr;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import lombok.SneakyThrows;
+import nix.education.util.SymbolsSolrQuery;
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.response.QueryResponse;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SolrBookAuthorSuggestionService implements SolrSuggestionService {
+    private final String authorSuggester = "authorSuggester";
```

Comment:
This is a constant. Please uppercase the field name

Reply:
Changed

Reply:
Changed.